test/provider/AppDirectory.unittest.ts

```diff
@@ -88,16 +96,35 @@ beforeEach(() => {
     });
 });
 
+beforeAll(async () => {
+    const store = new Store<ConfigurationObject>(require('../../gen/provider/config/defaults.json'));
+    Injector.rebind<'ENVIRONMENT'>(Inject.ENVIRONMENT).toConstantValue(createMockEnvironmnent() as Environment);
+    Injector.rebind<'CONFIG_STORE'>(Inject.CONFIG_STORE).toConstantValue({
+        config: store
+    });
+    Injector.rebind<'RESOLVER'>(Inject.RESOLVER).toConstantValue({
+        handleIntent: async (intent: Intent): Promise<ResolverResult> => {
+            return {app: null!};
+        },
+        cancel: async (): Promise<void> => {}
+    });
+
+    await Injector.init();
+});
+
 describe('AppDirectory Unit Tests', () => {
     describe('When querying the Directory', () => {
         beforeEach(() => {
             mockJson.mockResolvedValue(fakeApps);
```

Comment:
There's an inner beforeEach that also has this line. Are both needed?

Actually, could this move to `beforeEach` at the top of the file, and we get rid of an other `mockJson.mockResolvedValue(fakeApps);` lines in this file?

Reply:
done